timber-lint/src/main/java/timber/lint/TimberIssueRegistry.kt

```diff
@@ -7,7 +7,7 @@ import com.google.auto.service.AutoService
 
 @AutoService(value = [IssueRegistry::class])
 class TimberIssueRegistry : IssueRegistry() {
-  override val issues = WrongTimberUsageDetector.getIssues().asList()
+  override val issues = WrongTimberUsageDetector.issues.asList()
```

Comment:
```suggestion
  override val issues get() = WrongTimberUsageDetector.issues
```
Assuming you replace `arrayOf` with `listOf`.

Reply:
Technically the `api` and `minApi` should be using `get() = ` too... Oh well.

Reply:
updated